mozregression/build_range.py

```diff
@@ -35,7 +37,21 @@ def build_info(self):
         if self._build_info is None:
             try:
                 self._build_info = self._fetch()
-            except BuildInfoNotFound as exc:
+                if (
+                    isinstance(self._build_info, BuildInfo)
+                    and self._build_info.task_id
+                    and self._build_info.app_name == "gve"
+                ):
+                    # This is a taskcluster build, check if artifact is available.
+                    # See bug 1841685.
+
+                    # URL is an alias that redirects via a 303 status code.
+                    original_url = requests.head(self._build_info.build_url)
+                    redirect_url = original_url.headers["location"]
```

Comment:
To add some future-proofing it's probably worth handling the case where the URL doesn't redirect (eg. it returns a 403 directly and/or the `location` header doesn't exist)

Reply:
This is now handled properly with `allow_redirects=True`, and is also moved to a better place (to `fetch_build_info` instead of `build_range`).